dhcp_options.go

```diff
@@ -91,26 +91,43 @@ func (odbi *ovndb) dhcpOptionsAddImp(cidr string, options map[string]string, ext
 	return &OvnCommand{operations, odbi, make([][]map[string]interface{}, len(operations))}, nil
 }
 
-func (odbi *ovndb) dhcpOptionsSetImp(cidr string, options map[string]string, external_ids map[string]string) (*OvnCommand, error) {
-
-	row, err := newDHCPRow(cidr, nil, external_ids)
-	if err != nil {
-		return nil, err
+func (odbi *ovndb) dhcpOptionsSetImp(uuid string, options map[string]string, external_ids map[string]string) (*OvnCommand, error) {
+	row := make(OVNRow)
+	dhcpUUID := ""
+	dhcpuuidlist,err := odbi.DHCPOptionsList()
```

Comment:
why? you need to check only that this uuid exists, so don't over allocate slice for all options.
you already have uuid, so check for ok in odbi.cache[tableDHCPOptions][UUID]

Reply:
done